docs/src/api/class-consolemessage.md

```diff
@@ -5,6 +5,25 @@
 ## method: ConsoleMessage.args
 - returns: <[Array]<[JSHandle]>>
 
+List or arguments passed to a `console` function call.
+
+```js
```

Comment:
This js snippet feels very lonely without its Python, Java and C# friends.

Reply:
I found that I was about to replicate the docs from `Page.console`. Added a link to that. @yury-s might want to confirm a fix in the java docs.

I found this dotnet tool a few months ago https://github.com/SimonCropp/MarkdownSnippets. Although it's .NET only, the idea is straightforward. You code the code example on a `compilable place` such as a unit test. And the doc would point to that snippet. Then you have a process that would replace the snippet with that code.
Just for you to think about it.